samples/templates/default-azuredeploy-docker.json

```diff
@@ -392,7 +386,7 @@
                         "[if(variables('deployAppInsights'),concat('Microsoft.Insights/components/', variables('appInsightsName')),resourceId('Microsoft.KeyVault/vaults', variables('serviceName')))]"
                     ],
                     "properties": {
-                        "linuxFxVersion": "[concat('DOCKER|', variables('registryName'), '/', toLower(parameters('fhirVersion')), '_fhir-server',':', parameters('imageTag'))]",
+                        "linuxFxVersion": "[concat('DOCKER|', parameters('registryName'), '/', toLower(parameters('fhirVersion')), '-fhir-server',':', parameters('imageTag'))]",
```

Comment:
I think this will be a problem.

@LTA-Thinking thoughts if switch `'_fhir-server'` to 
`'-fhir-server'` as the default going forward to normalize the image names?